serve index.html on flask

bernie-graves opened the pull request:
I got index.html to be served by flask at 127.0.0.1:5000.
Then I tried to make requests to the API and was getting a response but it was 'No file part in the request' every time. I was able to successfully make requests from Postman but couldn't figure out how to get the file to the API using JS

bernie-graves committed:
Served index.html on flask

bernie-graves committed:
Update index.html

bernie-graves committed:
Update index.html

bernie-graves commented:
But there were no more CORS errors when making the requests which was nice

JasonFreeberg commented:
@bernie-graves let's talk tomorrow about this. The API expects a very particular request shape so I think we'll need to do a little digging on how to construct the request properly in JS

JasonFreeberg commented:
Before the meeting I'll try to pull this PR down and see what the client is sending

Commit:
Fix weird bug

https://stackoverflow.com/questions/39280438/fetch-missing-boundary-in-multipart-form-data-post

JasonFreeberg commented on .vscode/launch.json:
```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Debug Flask app",
+            "type": "python",
+            "request": "launch",
+            "python": "${fileWorkspaceFolder}/backend/image-upload-api/venv/Scripts/python.exe",
+            "program": "backend/image-upload-api/flask-api/main.py",
+            "args": [],
+            "jinja": true
+        }
+    ]
+}
```
@YiminArava4508 @harperklauke -- This is a configuration file for VS Code. It tells VS Code how to start the Flask app so that it can attach a debugger to it. Remind me in a future meeting to show you guys how to use the debugger, it can be super helpful

JasonFreeberg reviewed (approved):
Pushed a one-line fix for the bug where the client wouldn't get a response

Good to merge
